data_service/src/types.rs

```diff
@@ -1,5 +1,25 @@
+use chrono::{NaiveDateTime, TimeDelta};
+
 #[derive(PartialEq)]
 pub struct PuuidToChampionMapping {
     pub puuid: String,
     pub champion_name: String,
+    pub position: String,
+}
+
+#[derive(PartialEq)]
+pub struct MatchTimelineFrame {
+    pub mapping: PuuidToChampionMapping,
+    /**
+     * Offset of game start time
+     */
+    pub frame_time: TimeDelta,
```

Comment:
Should probably make sure you understand what this is. This should be 0 for the first frame

Reply:
should this be in seconds ( rounded to the nearest ofc )  or milliseconds?

Reply:
ms, please